meta/block.rb

```diff
@@ -184,6 +184,9 @@
   mutation 'foo.baz { }'
   mutation 'foo(&:bar).baz'
   mutation 'self.baz {}'
+  mutation 'foo(&nil).baz {}'
+  mutation 'foo(&self).baz {}'
```

Comment:
We could optionally skip `&nil` and `&self` since I think they are unlikely to yield alives but they are technically valid.

Reply:
I think this skip is valid and should be done in a followup. Does not block this PR.

Reply:
To elaborate on why: They are "static" expressions. The outcome of `&nil` and `&self` is known ahead of time to not be useful.